platform-includes/capture-error/apple.mdx

```diff
@@ -103,28 +103,49 @@ The SDK can't install the uncaught exception handler if a debugger is attached.
 
 </Note>
 
-By default, macOS applications do not crash whenever an uncaught exception occurs. To enable this with Sentry:
+By default, macOS applications do not crash whenever an uncaught exception occurs. As the Cocoa Frameworks are generally not [exception-safe on macOS](https://developer.apple.com/library/archive/documentation/Cocoa/Conceptual/Exceptions/Articles/ExceptionsAndCocoaFrameworks.html), we recommend this approach because the application could otherwise end up in a corrupted state.
```

Comment:
> we recommend this approach

it's not fully clear what "this" approach is - maybe "we recommend the approaches described below"?

Reply:
Is it clearer now, @kahest ?

Reply:
yep, thank you!

Reply:
Thanks for pointing it out. It was even a bit confusing to me after rereading it after a couple of hours.